### PR TITLE
Document async and MC QC operations

### PR DESCRIPTION
**Pull request by knopers8**

### CODE REVIEW

File: doc/Advanced.md [hunk 1]
```diff
@@ -644,6 +650,45 @@ The script `o2-qc-repo-move-objects` lets the user move an object, and thus all
...
+In such case, several masterjobs are run in parallel.
+Each produces a `QC.root` file which contains all the statistics for a masterjob.
+The last masterjob to complete recognizes this fact and merges all `QC.root` into `QC_fullrun.root` and only then uploads the results to QCDB.
+To find it, one can use `alien_find`:
+```
+> alien_find /alice/data/2022/LHC22m/523897/apass1_epn QC_fullrun.root
+/alice/data/2022/LHC22m/523897/apass1_epn/0750/QC/001/QC_fullrun.root
+```
+
+TODO explain how a connection to QCDB is made from Grid sites.
```
Comment (knopers8) [inline doc/Advanced.md:690]: @Barthelemy Could you please add a sentence or two explaining this?

Comment (Barthelemy) [reply doc/Advanced.md:690]: sure, will do

File: doc/Advanced.md [hunk 2]
```diff
@@ -312,9 +313,14 @@ o2-qc --config json:/${QUALITYCONTROL_ROOT}/etc/basic.json --remote-batch result
 Please note, that the local batch QC workflow should not work on the same file at the same time.
 A semaphore mechanism is required if there is a risk they might be executed in parallel.
 
-To be done:
-- merging multiple files into one, to allow for cases, when local batch workflows cannot access the same file.
-- support for Post-Processing.
+The file is organized into directories named after 3-letter detector codes and sub-directories representing Monitor Object Collections for specific tasks.
+To browse the file, one needs the associated Quality Control environment loaded.
```
Comment (JianLIUhep) [inline doc/Advanced.md:317]: Could you please elaborate a bit more about this? Does it mean checking the results for incomplete results? Maybe also add an example command?

File: doc/Advanced.md [hunk 3]
```diff
@@ -644,6 +650,45 @@ The script `o2-qc-repo-move-objects` lets the user move an object, and thus all
...
+Each produces a `QC.root` file which contains all the statistics for a masterjob.
+The last masterjob to complete recognizes this fact and merges all `QC.root` into `QC_fullrun.root` and only then uploads the results to QCDB.
+To find it, one can use `alien_find`:
+```
+> alien_find /alice/data/2022/LHC22m/523897/apass1_epn QC_fullrun.root
+/alice/data/2022/LHC22m/523897/apass1_epn/0750/QC/001/QC_fullrun.root
+```
+
+TODO explain how a connection to QCDB is made from Grid sites. 
+
```
Comment (JianLIUhep) [inline doc/Advanced.md:691]: I assume the instructions about the `aliqa1` machine would be added after it's migrated to a safer machine. or somewhere else? Correct me if I was wrong.

**Comment by JianLIUhep** [comment]
@knopers8 Thanks for writing these nice instructions. In general, everything is clear to me. Just added a couple of minor comments.

File: doc/Advanced.md (hunk 3 above)
Comment (knopers8) [reply doc/Advanced.md:691]: we have some internal documentation already, I will send it to you.

File: doc/Advanced.md (hunk 2 above)
Comment (knopers8) [reply doc/Advanced.md:317]: I will elaborate

**Pull request merged**